src/components/Register/Register.tsx

```diff
@@ -6,11 +6,37 @@ import TextInput from '../Input/TextBox/TextInput';
 import Button from '../ui/button/Button';
 import RegisterStore from '../../store/RegisterStore';
 import { t } from '../../i18n/i18n';
+import Validation from '../../lib/validation';
 
 const registerStore = new RegisterStore();
 
 @observer
 class Register extends React.Component {
+    registerHandler = () => {
+        let errorText = null;
```

Comment:
:String?

Reply:
question mark does not work. Only `:string` possible